src/JoomlaBrowser.php

```diff
@@ -268,7 +268,7 @@ public function installJoomlaMultilingualSite($languages = array())
 		foreach ($languages as $language)
 		{
 			$I->debug('I mark the checkbox of the language: ' . $language);
-			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]/input"]);
+			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]"]);
```

Comment:
It should be fine, I sometimes over think :stuck_out_tongue: . What you have sent is just perfect and more readable then my suggestion :+1: